manticore/ethereum.py

```diff
@@ -607,6 +603,19 @@ def _compile(source_code, contract_name):
             :return: name, source_code, bytecode, srcmap, srcmap_runtime, hashes
         """
         solc = "solc"
+
+        #check solc version
+        supported_versions = ('0.4.18', '0.4.21')
+        installed_version = check_output(["solc", "--version"])
```

Comment:
There is a variable called `solc` above which should be used here.

Anyway, as a long term solution we should probably have sth like `wrappers/solc.py` with e.g. `get_version`, `compile(source_code)` functions